<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player1_menuClass</class>
 <widget class="QWidget" name="player1_menuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>survivor.Q</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Century&quot;;
color: rgba(255, 255, 255, 100);
</string>
   </property>
   <property name="text">
    <string>&lt; BACK</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="coin">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>40</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/coin.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="coin_num">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>40</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Century&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QListView" name="background">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/gameMenu.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>710</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Century&quot;;</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="weapon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/weapon_player1.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="necklace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/necklace.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gloves">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/gloves.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chestpate">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/chestplate.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="belt">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/belt.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boot">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>470</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/boots.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>451</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 16pt &quot;Century&quot;;
color: rgba(255, 255, 255, 200);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="plainText">
    <string>Weapon

Level:1
ATK:20 (40)

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_store">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>580</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Century&quot;;
background-color:transparent;
color: rgba(255, 255, 255, 100);
</string>
   </property>
   <property name="text">
    <string>go to store to upgrade &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>270</width>
     <height>315</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/remenu/player1_big.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="orders">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Century&quot;;
background-color:transparent;
color: rgba(255, 255, 255, 100);</string>
   </property>
   <property name="text">
    <string>          press images to check the attributes</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>back</zorder>
  <zorder>coin</zorder>
  <zorder>coin_num</zorder>
  <zorder>start</zorder>
  <zorder>weapon</zorder>
  <zorder>necklace</zorder>
  <zorder>gloves</zorder>
  <zorder>chestpate</zorder>
  <zorder>belt</zorder>
  <zorder>boot</zorder>
  <zorder>textbox</zorder>
  <zorder>go_to_store</zorder>
  <zorder>player1</zorder>
  <zorder>orders</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="remenu.qrc"/>
 </resources>
 <connections/>
</ui>
